src/acquire.c

```diff
@@ -27,16 +27,16 @@ void acquire_process(acquire_t *st)
 {
     float complex max_v = 0;
     float angle, max_mag = -1.0f;
-    unsigned int samperr = 0, i, j;
+    unsigned int samperr = 0, i, j, keep;
     unsigned int mink = 0, maxk = FFTCP;
 
     if (st->idx != FFTCP * (M + 1))
         return;
 
-    if (st->ready && fabsf(st->slope) < 1 && st->samperr > 10)
+    if (st->input->sync.ready)
     {
-        mink = st->samperr - 10;
-        maxk = st->samperr + 10;
+        mink = FFTCP / 2 - 25;
+        maxk = FFTCP / 2 + 25;
```

Comment:
I had to widen the search range here to account for ppm error. Each invocation of `acquire_process` processes 64 * 2160 samples, so 100 ppm error causes a 14-sample offset between invocations.